src/StoreKeeper/WooCommerce/B2C/Helpers/KsesHelper.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Helpers;
+
+class KsesHelper
+{
+    public const ANCHOR = [
```

Comment:
this definition is already in `StoreKeeper/WooCommerce/B2C/Helpers/HtmlEscape::ALLOWED_ANCHOR`

Reply:
Removed KsesHelper and used HtmlEscape.